engine/src/main/java/com/redhat/cloud/notifications/processors/email/EmailAggregator.java

```diff
@@ -45,20 +44,12 @@ public class EmailAggregator {
     @Inject
     EmailSubscriptionRepository emailSubscriptionRepository;
 
-    @Inject
-    FeatureFlipper featureFlipper;
-
     // This is manually used from the JSON payload instead of converting it to an Action and using getEventType()
     private static final String EVENT_TYPE_KEY = "event_type";
     private static final String RECIPIENTS_KEY = "recipients";
 
-    @ConfigProperty(name = "notifications.get.aggregation.max.page.size", defaultValue = "10000")
-    int aggregationMaxPageSize;
-
-    private Set<String> getEmailSubscribers(EmailAggregationKey aggregationKey, EmailSubscriptionType emailSubscriptionType) {
-        return Set.copyOf(emailSubscriptionRepository
-                .getEmailSubscribersUserId(aggregationKey.getOrgId(), aggregationKey.getBundle(), aggregationKey.getApplication(), emailSubscriptionType));
-    }
+    @ConfigProperty(name = "notifications.aggregation.batch-size", defaultValue = "100")
```

Comment:
Thats more a page size than a batch size to me, because the size of the batch depends of number of aggregations in db.
This parameter is just limiting number of fetched records at once.